Source/Core/Core/State.cpp

```diff
@@ -190,6 +186,9 @@ static void DoState(PointerWrap& p)
   HW::DoState(p);
   p.DoMarker("HW");
   Movie::DoState(p);
+  if (SConfig::GetInstance().bWii)
+    Wiimote::DoState(p);
+  p.DoMarker("Wiimote");
```

Comment:
Is there any reason why you moved this down here?
Also, it seems you put it a line too low or a line too high; since the Movie-marker should be next to `Movie::DoState(p)`.

Reply:
Oops. But yeah. Wiimote state has to be synced after HW for a reconnection to not get ignored by the in-progress save-state load.

Reply:
Fixed the marker.